skema/program_analysis/CAST/cast_walker.py

```diff
@@ -0,0 +1,59 @@
+import json
+import os.path
+from pathlib import Path
+from typing import Any, Dict, List, Union
+
+from tree_sitter import Tree, Language, Parser, Node
+
+from skema.program_analysis.CAST2FN.cast import CAST
+from skema.program_analysis.CAST2FN.model.cast import (
+    Module,
+    SourceRef,
+    Assignment,
+    LiteralValue,
+    Var,
+    VarType,
+    Name,
+    Operator,
+    AstNode,
+    SourceCodeDataType,
+    ModelImport,
+    FunctionDef,
+    Loop,
+    Call,
+    ModelReturn,
+    ModelIf,
+    RecordDef,
+    Attribute,
+)
+
+
+# Generate a CAST for matlab and visit every node
+
+LANGUAGE_LIBRARY_REL_PATH = 'tree-sitter-matlab' # TODO derive
```

Comment:
@vincentraymond-ua , how do you recommend retrieving this?

Reply:
You can import the shared object file path directly from build parsers and use it like so:

1. Replace line 33  with `from skema.program_analysis.tree_sitter_parsers.build_parsers import INSTALLED_LANGUAGES_FILEPATH`
2. Replace line 46 with `INSTALLED_LANGUAGES_FILEPATH`

Reply:
That file has been deleted, the functionality was subsumed into CAST/matlab/matlab_to_cast.py